src/devices/tuya.ts

```diff
@@ -11603,6 +11603,87 @@ const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS0601',
+                manufacturerName: '_TZE204_ex3rcdha',
+            },
+        ],
+        model: 'TS0601',
```

Comment:
Please use the model number that is written on the device here.

Reply:
Yes it is, I will modify the model accordingly